docs/Quickstart.md

```diff
@@ -26,15 +26,6 @@ Head to the Heroku dashboard and create a new app, or run the following commands
 CLI:
 ```sh
 $ heroku create name
-$ heroku git:remote -a name
```

Comment:
Heroku does this by default, it shouldn't be necessary to do manually.

Reply:
(also, the [video](https://www.youtube.com/watch?v=yGxeoAHlQOg) doesn't have this step).